packages/agents/docs/core/context.md

```
@@ -0,0 +1,156 @@
+  signal: controller.signal,
+});
+
+// Cancels all in-flight operations
+controller.abort();
+```
+
+### Logger Scoping
+
+The framework creates child loggers at each scope boundary using `log.child()`. This means log output automatically includes execution context without manual threading:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)